test/integration/api/token_test.rb

```diff
@@ -1,4 +1,4 @@
-require 'test_helper'
+require_relative '../../test_helper'
```

Comment:
As you probably noticed i prefer `require 'test_helper'`. Maybe we have to agree to disagree here. I like to be able to move tests into nested directories without having to touch them.

I run tests either with `ruby -Itest test/integration/api/tokent_test.rb` or with `rake TEST=test/integration/tokent_test.rb`. The former adds the test dir to the load path and the latter has the benefit of using spring (like zeus) in rails4 - and the disadvantage of breaking tabcompletion for the test file name. I guess i will have to write a little test script at some point that gives me both.